components/BlogPost/blogpost.tsx

```diff
@@ -0,0 +1,84 @@
+import ComposedTitles from "@components/Common/ComposedTitles";
+import PrimaryDescription from "@components/Common/PrimaryDescription";
+import { blogposts } from "@lib/consts";
+import Image from "next/image";
+import BreadCrumbs from "../Common/BreadCrumbs/BreadCrumbs";
+
+/* eslint-disable @next/next/no-img-element */
+const blogpost = () => {
+  const {
+    title,
+    authorName,
+    date,
+    subTitle,
+    content,
+    authorImage,
+    imageWidth,
+    imageUnit,
+    imageHeight,
+    contentImage,
+  } = blogposts;
```

Comment:
@oscarcorellan The name of the variable blogposts should be on camelCase 
```suggestion
  } = blogPosts;
```